sample-apps/react-native/dogfood/package.json

```diff
@@ -32,7 +32,7 @@
     "@stream-io/flat-list-mvcp": "^0.10.3",
     "@stream-io/react-native-webrtc": "118.0.1",
     "@stream-io/video-react-native-sdk": "workspace:^",
-    "react": "18.2.0",
+    "react": "18.3.1",
```

Comment:
Same here: @khushal87 @santhoshvai

Reply:
we need to keep it pinned in apps for sure, as RN version is tied to React..

for example see the [latest React conference app](https://github.com/expo/react-conf-app/blob/bb9016a22c95e11a3a8682ee9e3c6f1758f98f15/package.json#L47)